lib/bundle/brew_installer.rb

```diff
@@ -9,11 +9,18 @@ def initialize(name, options = {})
         @full_name = name
         @name = name.split("/").last
         @args = options.fetch(:args, []).map { |arg| "--#{arg}" }
+        @restart = options.fetch(:restart_service, false)
       else
         raise "Unable to install #{name}. Homebrew is not currently installed on your system"
       end
     end
 
+    def run
+      install_or_upgrade && restart
```

Comment:
I fix other comment but can you explain more why you will add the (optional) `restart` into `install_or_upgrade`. Because currently i fetch it from the init of `BrewInstaller`. 
Moreover, i think use a boolean parameter in a method to control logic flow should be avoided.